### ISSUE TITLE
NOTE: THIS REPO IS OFICIALLY DEAD & ORPHANED !

### ISSUE THREAD

**Issue by devZer0** [dailab/libsml#14]
Please go to: https://github.com/volkszaehler/libsml

For reason, have a look at https://demo.volkszaehler.org/pipermail/volkszaehler-dev/2015-January/004064.html (german text)

Translated with www.DeepL.com/Translator

Servus,

after a bit of initiative and mail requests from Roland to Juri and the Dai-Lab it turned out that libsml (an important base component for the census project) is currently "unmaintained", i. e. nobody cares about the further development of libsml (as it has been in the last 2 years).

Adding even small but helpful changes like "make install"support (PR by Peter Evertz, also here on the list: https://github.com/peterevertz/libsml/commit/d7ab04aed281e8bfed20f678f39249b1a9a1e213) has been pending for almost two years now, but this has failed due to the fact that Juri is no longer working in the Dai-Lab and has no more commit privileges on the repository, and that the priorities are elsewhere in the Dai-Lab and the actual successor of Juri does not take on the role of maintainer.

We, i. e. Juri, Roland and I have come to the conclusion that it makes sense to fork libsml and hang it within the census project. As a prerequisite for this, however, I see that there are also people here who are involved - otherwise we will only be able to create the next siding here.

Roland himself would in principle be willing to take over the position as libsml-maintainer, but says that he does not have enough experience with SML, C and Git for maintaining a not uncomplicated C-software. In short, he would feel a little overwhelmed on his own.

Would people on this list be willing to take over or at least carry the libsml care?

This means that we are looking for people who would either take over the maintenance of the library completely or who could at least provide technical advisory board and support to help Roland to bring libsml "forward".

In short: now is the time to implement the New Year's resolutions; -)

Thanks again to Roland for his commitment!

Greetings, J.

**Comment by devZer0** [comment]
i'd appreciate if someone with commit rights to dailab/libsml could change README.md appropriately, as this message may be overseen...

**Comment by devZer0** [comment]
it really sucks that this repo is still being found as the "default" one when searching on google for libsml
